build(napi/oxlint): do not fully minify JS code

overlookmotel opened the pull request:
Alter TSDown settings for `napi/oxlint2`.

We intended the code to be minified, but not to have symbols mangled, or whitespace removed, so it remains somewhat readable. It seems that TSDown may have changed how it handles these options, as `minify: { compress: true }` now turns on mangling and whitespace removal.

Alter the TSDown options to get the intended output.

camc314 commented:
### Merge activity

* **Sep 10, 5:48 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Sep 10, 5:48 PM UTC**: `camc314` added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc).

overlookmotel committed:
build(napi/oxlint): do not fully minify JS code (#13661)

Alter TSDown settings for `napi/oxlint2`.

We intended the code to be minified, but not to have symbols mangled, or whitespace removed, so it remains somewhat readable. It seems that TSDown may have changed how it handles these options, as `minify: { compress: true }` now turns on mangling and whitespace removal.

Alter the TSDown options to get the intended output.
